static/js/portico/signup.js

```diff
@@ -4,7 +4,9 @@ $(function () {
 
     $.validator.addMethod('password_strength', function (value) {
         return common.password_quality(value, undefined, $('#id_password, #id_new_password1'));
-    }, "Password is too weak.");
+    }, function () {
+        return common.password_warning($('#id_password, #id_new_password1').val(), $('#id_password, #id_new_password1'));
```

Comment:
This code seems a bit strange to me, like it should be the default somehow.  At any rate, you might want to extract a local variable so you don't need to repeat the same selector twice.